tests/test.sh

```diff
@@ -5,10 +5,14 @@
 #
 set -e
 
-sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
-sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io
+if [ -z ${NO_APT_INSTALL+x} ]; then
```

Comment:
why do we need it if it's always disabled?

Reply:
@YoniKF Added this feature so people can simply run this script and it will set up the whole environment for them.